vertx-web-graphql/src/main/java/io/vertx/ext/web/handler/graphql/impl/GraphQLHandlerImpl.java

```diff
@@ -305,7 +305,7 @@ private void executeOne(RoutingContext rc, GraphQLQuery query) {
 
   private String getContentType(RoutingContext rc) {
     String contentType = rc.request().headers().get(HttpHeaders.CONTENT_TYPE);
-    return contentType == null ? "application/json" : contentType.toLowerCase();
+    return contentType == null ? "application/json" : contentType.toLowerCase().split(";")[0];
```

Comment:
@cristianonicolai actually, you could replace the method body with:

```java
    String contentType = rc.parsedHeaders().contentType().value();
    return contentType.isEmpty() ? "application/json" : contentType.toLowerCase();
```

Because vertx-web parses the header already.

Reply:
I went ahead and merged. I'll do the update.

Reply:
@tsegismont ok, thanks